### PR TITLE
24Q2

### PR DESCRIPTION
**Pull request by dariok**

### CODE REVIEW

**Comment by WunschK** [comment]
lgtm

File: edoc/modules/error.xqm
Comment (WunschK) [inline edoc/modules/error.xqm:1]: shouldn't "wdb0000" be removed, as "wdbErr:wdb0000" already covers this? (in the new version)

File: eslint.config.js
Comment (WunschK) [inline eslint.config.js:1]: mistake or was your intention to share a linting-config?

**Review by WunschK — CHANGES_REQUESTED** [review]
linting-config shared accidentally
wdb0000 without wdbErr should be removed from error.xqm

**Comment by dariok** [comment]
- ESLint config: should be helpful to assure using the same standards and checks; this contains no secrets
- Names of errors may be with or without a namespace
- wherever possible, similar errors should have a similar error number

**Review by WunschK — APPROVED** [review]

**Pull request merged**